dex-it-tools/src/main/scala/com/wavesplatform/dex/it/json/package.scala

```diff
@@ -2,14 +2,16 @@ package com.wavesplatform.dex.it
 
 import com.wavesplatform.common.state.ByteStr
 import com.wavesplatform.common.utils.EitherExt2
-import com.wavesplatform.transaction.assets.exchange.AssetPair
+import com.wavesplatform.transaction.assets.exchange
+import com.wavesplatform.transaction.assets.exchange.{AssetPair, Order}
 import com.wavesplatform.transaction.{Asset, Transaction, TransactionFactory}
 import play.api.libs.json._
 
 import scala.util.{Failure, Success}
 
 package object json {
-  implicit val byteStr: Format[ByteStr] = Format(
+
+  implicit val byteStrFormat: Format[ByteStr] = Format(
```

Comment:
There can't be `byteStr` which is not `Format` or `Reads` or `Writes` here, so an original name has less factor of duplication:

> byteStr**Format**: **Format**[ByteStr] = **Format**(

vs

> byteStr: **Format**[ByteStr] = **Format**(

IMO an additional word **format** doesn't add value here. Same idea for renamings like `r` (which means "result") to `aLooongNameLikeAFunctionNameButWeKnowWhatFunctionReturns` ( :-) ). Mostly it's just a matter of taste.

But you don't need to change names again.